packages/amplify-cli/src/commands/build-override.ts

```diff
@@ -0,0 +1,56 @@
+import { $TSContext, IAmplifyResource } from 'amplify-cli-core';
+import { printer } from 'amplify-prompts';
+/**
+ * Command to transform CFN with overrides
+ */
+const subcommand = 'build-override';
+
+export const run = async (context: $TSContext) => {
+  let resourceName = context?.input?.subCommands?.[0];
+  const categoryName = context?.input?.subCommands?.[1];
+
+  if (categoryName === undefined) {
+    // if no category is mentioned , then defaults to all resource
+    resourceName = undefined;
+  }
+
+  try {
+    const resourcesToBuild: IAmplifyResource[] = await getResources(context);
+    let filteredResources = {};
+    if (categoryName) {
+      filteredResources = resourcesToBuild.filter(resource => resource.category === categoryName);
+    }
+    if (categoryName && resourceName) {
+      filteredResources = resourcesToBuild.filter(resource => resource.category === categoryName && resource.resourceName === resourceName);
+    }
+    for (const resource of resourcesToBuild) {
+      await context.amplify.invokePluginMethod(context, 'awscloudformation', undefined, 'transformCfnWithOverrides', [context, resource]);
+    }
+  } catch (err) {
+    printer.error(err.stack);
+    printer.error('There was an error building the resource');
+    context.usageData.emitError(err);
+    process.exitCode = 1;
+  }
+};
+
+const getResources = async (context: $TSContext): Promise<IAmplifyResource[]> => {
+  const resources: IAmplifyResource[] = [];
+  const { resourcesToBeCreated, resourcesToBeUpdated } = await context.amplify.getResourceStatus();
```

Comment:
This also returns allResources AFAIK - can we use that instead?

Reply:
Why do we need all resources , we only need the resources created / updated as overrides will only be applied on these